analyzers/tests/SonarAnalyzer.TestFramework/Build/ProjectBuilder.cs

```diff
@@ -47,7 +47,10 @@ public Compilation GetCompilation(ParseOptions parseOptions = null, CompilationO
     }
 
     public Document FindDocument(string name) =>
-        project.Documents.Single(d => d.Name == name);
+        project.Documents.SingleOrDefault(d => d.Name == name);
```

Comment:
It means that now this method (and the following one) can return `null`, did you check if it's contemplated on the caller side? Do we have test converage for this?

Reply:
That was legacy that I should have removed 😅